ui/tests/integration/helpers/date-format-test.js

```diff
@@ -48,7 +48,7 @@ module('Integration | Helper | date-format', function (hooks) {
     assert.dom(this.element).includesText(format(formattedDate, 'MMMM dd, yyyy hh:mm:ss a'));
   });
 
-  test('displays time zone if withTimeZone=true', async function (assert) {
+  skip('displays time zone if withTimeZone=true', async function (assert) {
```

Comment:
Is it worth updating this to the current version of the test rather than skipping it?

Reply:
I noticed that this test on `main` is not skipped so I was referring to copying it over to this branch. I'm guessing it was flaky before and was updated but not backported.